backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java

```diff
@@ -91,6 +94,15 @@ public Facility(Organization org, String facilityName, String cliaNumber,
 		this.telephone = phone;
 	}
 
+	public Facility(Organization org, String facilityName, String cliaNumber,
```

Comment:
Am I right in thinking that the other fat constructor is now used only for tests? If so, we should update the tests to use the new one, and not have this many constructors.

Reply:
Correct, these are only used in the tests, so I changed all tests to use the same fat constructor and got rid of the others :+1:

Reply:
Hmmm, looks like Sonar is not happy about these fat constructors:
`Constructor has 9 parameters, which is greater than 7 authorized.`
Should I just make a constructor with 7 parameters instead to avoid the warning?

Reply:
No, Sonar can stuff it. 😁 

Seriously though, if we can find ways to use business objects to group arguments (e.g. phone and e-mail, as noted elsewhere) that would be a nice way to cut down on it, but that's not specific to this change—the too-many-parameters problem is the code smell that our entire codebase smells of, and I'd rather have that and know that it's a problem than have a bunch of silly workarounds that serve only to mask the underlying problem.